web/templates/popovers/navbar/navbar_personal_menu_popover.hbs

```diff
@@ -115,6 +115,34 @@
                         <span class="slider"></span>
                     </div>
                 </li>
+                <li role="none" class="popover-menu-list-item">
+                    <div class="info-density-controls">
+                        <div class="font-size-button-group button-group" data-property="web_font_size_px">
+                            <span class="current-value" hidden="true">{{web_font_size_px}}</span>
+                            <button class="info-density-button decrease-button">
+                                <i class="zulip-icon zulip-icon-minus" aria-hidden="true"></i>
+                            </button>
+                            <button class="info-density-button default-button">
+                                <i class="zulip-icon zulip-icon-type-big" aria-hidden="true"></i>
+                            </button>
+                            <button class="info-density-button increase-button">
+                                <i class="zulip-icon zulip-icon-plus" aria-hidden="true"></i>
+                            </button>
+                        </div>
+                        <div class="line-height-button-group button-group" data-property="web_line_height_percent">
+                            <span class="current-value" hidden="true">{{web_line_height_percent}}</span>
+                            <button class="info-density-button decrease-button">
+                                <i class="zulip-icon zulip-icon-minus" aria-hidden="true"></i>
+                            </button>
+                            <button class="info-density-button default-button">
+                                <i class="zulip-icon zulip-icon-line-height-big" aria-hidden="true"></i>
+                            </button>
+                            <button class="info-density-button increase-button">
+                                <i class="zulip-icon zulip-icon-plus" aria-hidden="true"></i>
+                            </button>
```

Comment:
I think we need `aria-label` and tooltips on these increase/decrease buttons for accessibility.

Reply:
We have tooltips already that are added in `tippyjs.ts`. And for `aria-label`, can we show the values like we show in tooltips dynamically? Currently, I have set it to texts like "Increase font size", "Reset font size to default", etc.